android/build.gradle

```diff
@@ -2,9 +2,9 @@ group 'com.airship.airship'
 version '1.0-SNAPSHOT'
 
 buildscript {  
-    ext.kotlin_version = '1.5.31'
-    ext.coroutine_version = '1.5.0'
-    ext.airship_version = '16.4.0'
+    ext.kotlin_version = '1.7.0'
```

Comment:
we haven't bumped Kotlin in the Android SDK yet, so it'd probably be best to keep kotlin on `1.5.31` and bump coroutines to `1.5.2` to match for now